app/views/promotion/newspaperLandingPage.scala.html

```diff
@@ -48,7 +48,7 @@
         @fragments.page.header(title)
 
         <section class="promotion-description">
-            <p>@promotion.description</p>
+            <p>@promotion.landingPage.description.filter(_.isEmpty).getOrElse(promotion.description)</p>
```

Comment:
it's an option of string, but that string might itself be empty - can we resolve that when it goes into the promotion object rather than flattening it out at the point of use?